feature.js

```diff
@@ -189,8 +192,7 @@
     // Tests if touch events are supported, but doesn't necessarily reflect a
     // touchscreen device
     touch: !!(('ontouchstart' in window) ||
-      window.navigator &&
-      window.navigator.msPointerEnabled &&
+      nav.msPointerEnabled &&
```

Comment:
I think we still need to first test the presence of “nav” here, otherwise this might in some cases throw ugly error and stop executing any further scripts. so… I think this line should be `nav && nav.msPointerEnabled`.

Reply:
oh but wait. Now we’re fallbacking to empty object. Right. In that case you can possibly ignore more previous comment.

Reply:
Yeah, I made sure I did a bit of testing here. If we're setting `nav` to `window.navigation || {}` then `nav.msPointerEnabled` would not throw an error, but return `undefined` which is falsy and will cause the condition to return false.